@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -691,12 +691,32 @@ export interface MarkerTechniqueParams extends BaseTechniqueParams {
     wrappingMode?: DynamicProperty<"None" | "Character" | "Word">;
     /**
      * Text position regarding the baseline.
+     *
+     * @deprecated Use [[hPlacement]] attribute, which defines label text placement in exactly
+     * opposite way, text that were `Left` aligned are by deduction `Right` placed relative to
+     * the label icon.
      */
     hAlignment?: DynamicProperty<"Left" | "Center" | "Right">;
     /**
      * Text position inside a line.
+     *
+     * @deprecated Use [[vPlacement]] attribute instead.
```

Comment:
I think we should not deprecate it. Ultimately we could support combinations of placement and alignment (or to go really crazy arrays of placements and alignments).
Not sure if anybody needs it though

Reply:
I don't know either, but this is what we agreed on the last standup :). I can quickly get rid of depreciation of course, but the real problem is what should I actually implement in this ticket :).

Reply:
@ninok @kkostecki Instead of deprecating alignment, we could just document the current limitation (that hAlignment is taken as the opposite value of hPlacement).
Then in the future we could add support for combinations.

But just to be on the same page, I only see the point of specifying alignment together with placement for multiline text right? Otherwise, a I don't see with respect to what would you align a left-align left-placed line of text.

Reply:
I agree, maybe better to leave with both... not deprecated, and then we have the possibility to extend... actually the second PR #1407 was directed that way, but I wanted to still be on AC side so there are lines that override alignment with placement (well commented).
Anyway @atomicsulfate sorry for taking you time, but this approach will be probably all gone and replaced with placement: "TR" | "R" | "BR" ... attributes, possibly with priority array as proposed in ticket comments (see Jira).